library/Icingadb/Widget/ItemList/StateListItem.php

```diff
@@ -80,6 +80,9 @@ protected function assembleVisual(BaseHtmlElement $visual)
         if ($this->state->is_handled) {
             $stateBall->addHtml(new Icon($this->getHandledIcon()));
             $stateBall->getAttributes()->add('class', 'handled');
+        } elseif ($this->state->getStateText() === 'pending' && $this->state->in_downtime) {
+            $stateBall->addHtml(new Icon($this->getHandledIcon()));
+            $stateBall->getAttributes()->add('class', 'handled');
```

Comment:
I'm not sure this class suits hollow state balls, the contrast of the light outline with the background is very low. In light mode of course.

@flourish86 What do you think?

Reply:
> @flourish86 What do you think?

I see your point. After looking at the following test view (screenshot), I find that it works better to not apply the transparency.

We don't have any other outlined state balls that could be handled, do we? If so, I think we can establish the rule of not reducing transparency on handled item state balls. The icon itself is still enough to convey the handled state.

![Screen Shot 2022-08-17 at 16 43 44](https://user-images.githubusercontent.com/6977434/185169823-906caa41-afdb-4300-8d1c-5815d2204a61.png)

Reply:
Aye, thanks :heart:. @raviks789 Please don't add the handled class here.